### ISSUE TITLE
Any plans to setup as a Bazel workspace?

### ISSUE THREAD

**Issue by raliste** [googleapis/google-cloud-java#3416]

**Comment by hzyi-google** [comment]
Hi @raliste , are you referring to using bazel for building the project instead of maven?

**Comment by raliste** [comment]
Yes @hzyi-google . That would help projects that depend on this repo to use Bazel as well (without maintaining third_party BUILDs)
